### PR TITLE
RSDK-2399 Removing Pre-Map Representation Endpoints

### PR DESCRIPTION
**Pull request by jeremyrhyde**
This PR removes the old endpoints from the SLAM proto to make room for the final versions:
- `GetPosition` (unary)
- `GetPointCloudMap` (stream)
- `GetInternalState` (stream)

JIRA Ticket: https://viam.atlassian.net/browse/RSDK-2239

Blocking PRs:
- https://github.com/viamrobotics/rdk/pull/2014
- https://github.com/viamrobotics/viam-cartographer/pull/10

### CODE REVIEW

**Review by JohnN193 — APPROVED** [review]

**Review by tessavitabile — APPROVED** [review]
Jeremy, can you confirm that this is ready for me to merge?

**Comment by jeremyrhyde** [comment]
> Jeremy, can you confirm that this is ready for me to merge?

Just waiting on sign off from Nick otherwise all PRs that were blocking this have been merged and we should be good to go!

File: service/sensors/v1/sensors.pb.gw.go [hunk 1]
```diff
@@ -15,7 +15,7 @@ import (
 
 	"github.com/grpc-ecosystem/grpc-gateway/v2/runtime"
 	"github.com/grpc-ecosystem/grpc-gateway/v2/utilities"
-	"go.viam.com/api/common/v1"
```
Comment (nicksanford) [inline service/sensors/v1/sensors.pb.gw.go:18]: why did this change?

File: service/shell/v1/shell.pb.gw.go [hunk 2]
```diff
@@ -15,7 +15,7 @@ import (
 
 	"github.com/grpc-ecosystem/grpc-gateway/v2/runtime"
 	"github.com/grpc-ecosystem/grpc-gateway/v2/utilities"
-	v1_0 "go.viam.com/api/common/v1"
```
Comment (nicksanford) [inline service/shell/v1/shell.pb.gw.go:18]: Why did this change?

File: component/servo/v1/servo.pb.gw.go [hunk 3]
```diff
@@ -15,7 +15,7 @@ import (
 
 	"github.com/grpc-ecosystem/grpc-gateway/v2/runtime"
 	"github.com/grpc-ecosystem/grpc-gateway/v2/utilities"
-	"go.viam.com/api/common/v1"
```
Comment (nicksanford) [inline component/servo/v1/servo.pb.gw.go:18]: why did this change?

**Review by nicksanford — CHANGES_REQUESTED** [review]
@jeremyrhyde why did a bunch of non slam files change?

File: service/sensors/v1/sensors.pb.gw.go (hunk 1 above)
Comment (jeremyrhyde) [reply service/sensors/v1/sensors.pb.gw.go]: Not really sure, but I will confirm. I do note multiple PRs over the past few days have switched it to v`1_0` to `v1` or vica versa

**Pull request closed without merging**